src/coreclr/src/inc/corpriv.h

```diff
@@ -212,6 +212,16 @@ DECLARE_INTERFACE_(IMetaDataEmitHelper, IUnknown)
 
 };  // IMetaDataEmitHelper
 
+//////////////////////////////////////////////////////////////////////////////
+// enum CorElementTypeMulticoreJitSig defines some additional internal ELEMENT_TYPE's
+// values that are only used by MulticoreJitSig signatures.
+//////////////////////////////////////////////////////////////////////////////
+typedef enum CorElementTypeMulticoreJitSig
+{
+    ELEMENT_TYPE_MODULE_MULTICOREJITSIG  = 0x3a,     // multicorejit encoding for external module id#
```

Comment:
Why not reuse `ELEMENT_TYPE_MODULE_ZAPSIG` for this one too?

You should be able to use `ZapSigContext` to interpret the index as appropriate.

Reply:
Thank you. I will update.